expectRetrying.ts

```diff
@@ -352,6 +363,47 @@ export function createExpectation(
       );
     },
 
+    async toBeEmpty(
+      options: Partial<RetryConfig> = retryConfig,
+    ): Promise<void> {
+      await createMatcher(
+        "toBeEmpty",
+        async () => {
+          try {
+            // First check if the element is an input, textarea or select.
+            return await locator.inputValue().then((text) => text.length === 0);
+          } catch (error) {
+            let msg = "";
+            if (error instanceof Error) {
+              msg = error.toString();
+            } else {
+              // Errors from k6 are not instances of Error at the moment.
+              msg = String(error);
+            }
+
+            if (
+              !msg.includes(
+                "Node is not an <input>, <textarea> or <select> element",
```

Comment:
Is this always the message returned when you call `inputValue()` over a locator that doesn't point to an input element? 🤔

Reply:
Yeah, this is a bit smelly 😭 The error message shouldn't change. I think to ensure that we don't break it, we will need a test in k6. Since the plan is to move this to k6 then maybe we will be able to catch any issues early.

Maybe we should implement a `isEmpty` method in k6 browser so that it handles the logic in there 🤔 I'd prefer to do this later though.

Reply:
> Also, is this is another case of k6 not being able to return error types that can be instanceof(MyErrorType) again?

Sounds like this is common. Is it expected? Have we documented it? Perhaps it's the moment to start.

Reply:
> Also, is this is another case of k6 not being able to return error types that can be instanceof(MyErrorType) again?

I'm not aware of any issue in k6 (I can't find anything). The closest one is this one: https://github.com/grafana/k6/issues/4714.

Reply:
Yeah, I'd suggest working on this as soon as possible. And, if not (would take longer than desired), then I'd suggest allocating some time to properly identify what's the root cause of such trouble and design a larger/more general solution for this type of scenarios.

Reply:
I've added that issue to be considered for the next release cycle.

Reply:
Thanks! 🙌🏻